entries/m/morganstanley.com.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Morgan Stanley": {
+    "domain": "morganstanley.com",
+    "url": "https://www.morganstanley.com/",
```

Comment:
A URL tag isn't needed if the domain (`https://morganstanley.com/`) redirects to the same page as the value of the URL tag.
```suggestion
```